src/components/ImageHeader.js

```diff
@@ -3,12 +3,12 @@
 import React from "react";
 
 type ImageHeaderProps = {
-	imageSrc: string,
-	style?: any
+ imageSrc: string,
+ style?: any
 };
 
 const defaultStyles = {
-  possition: "relative"
+  possition: "relative",
```

Comment:
@stan-sack Can we please remove the comma on the last element as well

Reply:
Done. Also fixed a typo (possition should be position). This may change the behaviour of the component though. Do you want me to leave it in?

Reply:
@stan-sack Nah, it's a typo and should be fixed. If the behaviour is changed I'm going to fix it.